lineman/app/models/user_records_interface.coffee

```diff
@@ -5,6 +5,8 @@ angular.module('loomioApp').factory 'UserRecordsInterface', (BaseRecordsInterfac
     constructor: (recordStore) ->
       @baseConstructor recordStore
       @profileClient = new RestfulClient 'profile'
```

Comment:
@gdpelican I think this will fix a bug for you in profile picture update land. (this also helped convince me that the subclassing restfulclient and removing recordinterfaces approach feels right)